android-resources/www/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+  <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.find,Number.isFinite,Number.isInteger,console,console.log,document.head,performance.now"></script>
+  <head>
+    <title>CodeWorld</title>
+    <style type="text/css">
```

Comment:
I noticed a lot of JavaScript and CSS is copied and pasted from other files.  Is that because you can't package those other files into the Android apk?  That would be a shame.  If possible, let's try to use the same code rather than copy.